src/tools/DemandGenerator.cpp

```diff
@@ -93,8 +98,8 @@ Demand * DemandGenerator::generateSinglePerturbatedDemand(bool checkFeasibility)
 		// create the first week
 		Demand* pCompleteDemand = demandHistory_[indexInHistory[0]]->randomPerturbation();
 
-		int t = demandHistory_.size();
-		int i = indexInHistory[0];
+		// int t = demandHistory_.size();
+		// int i = indexInHistory[0];
```

Comment:
Delete?

Reply:
done in  6efc95cf3ddeb8d97e69f945b603238e901ba853